staging/src/k8s.io/apiserver/pkg/util/webhook/metrics.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webhook
+
+import (
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+var x509MissingSANCounter = metrics.NewCounterVec(
+	&metrics.CounterOpts{
+		Subsystem:      "webhooks",
+		Name:           "x509_missing_san_count",
+		Help:           "Counts the number of connection failures dues to the lack of x509 certificate SAN extension missing",
```

Comment:
we will set the GODEBUG=... var in the base image. So this is not about actual failures. But about failures that would happen if we did not set GODEBUG.

Reply:
The server cert SAN check is to be found https://github.com/kubernetes/kubernetes/pull/95396/files#diff-e83399497d7ae943bd4bf530c2fbe509R64-R69, I need to update this string